gp-templates/translation-row-editor-meta-status.php

```diff
@@ -11,20 +11,25 @@
-					<button class="button is-small approve" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-current_' . $translation->id ) ); ?>" title="<?php esc_attr_e( 'Approve this translation. Any existing translation will be kept as part of the translation history.', 'glotpress' ); ?>"><strong>+</strong> <span><?php _ex( 'Approve', 'Action', 'glotpress' ); ?></span></button>
-				<?php endif; ?>
-				<?php if ( 'rejected' !== $translation->translation_status ) : ?>
-					<button class="button is-small reject" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-rejected_' . $translation->id ) ); ?>" title="<?php esc_attr_e( 'Reject this translation. The existing translation will be kept as part of the translation history.', 'glotpress' ); ?>"><strong>&minus;</strong> <?php _ex( 'Reject', 'Action', 'glotpress' ); ?></button>
-				<?php endif; ?>
-				<?php if ( 'fuzzy' !== $translation->translation_status ) : ?>
+		<?php if ( ! ( ( 'changes_requested' == $translation->translation_status ) && ( ! apply_filters( 'gp_enable_changes_requested_status', false ) ) ) ) : // todo: delete when we merge the gp-translation-helpers in GlotPress. ?>
+			<?php if ( $translation->translation_status ) : ?>
+				<?php if ( $can_approve_translation ) : ?>
+					<?php if ( 'current' !== $translation->translation_status ) : ?>
+						<button class="button is-small approve" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-current_' . $translation->id ) ); ?>" title="<?php esc_attr_e( 'Approve this translation. Any existing translation will be kept as part of the translation history.', 'glotpress' ); ?>"><strong>+</strong> <span><?php _ex( 'Approve', 'Action', 'glotpress' ); ?></span></button>
+					<?php endif; ?>
+					<?php if ( ( 'rejected' !== $translation->translation_status ) && ( 'changes_requested' !== $translation->translation_status ) ) : ?>
+						<button class="button is-small reject" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-rejected_' . $translation->id ) ); ?>" title="<?php esc_attr_e( 'Reject this translation. The existing translation will be kept as part of the translation history.', 'glotpress' ); ?>"><strong>&minus;</strong> <?php _ex( 'Reject', 'Action', 'glotpress' ); ?></button>
+						<?php if ( apply_filters( 'gp_enable_changes_requested_status', false ) ) : // todo: delete when we merge the gp-translation-helpers in GlotPress ?>
+							<button class="button is-small changes_requested" style="display: none;" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-changes_requested_' . $translation->id ) ); ?>" title="<?php esc_attr_e( 'Request changes for this translation. The existing translation will be kept as part of the translation history.', 'glotpress' ); ?>"><strong>&minus;</strong> <?php _ex( 'Request changes', 'Action', 'glotpress' ); ?></button>
```

Comment:
I think we should not use underscores in css classes for consistency reasons.

Reply:
I have updated these texts: `changes_requested` to `changesrequested` and `request_changes` to `requestchanges`. Done here https://github.com/GlotPress/GlotPress/pull/1451/commits/da177763d4f85689b0d636238eec37e1a2167bb5.